translations/fr.json

```diff
@@ -163,10 +159,9 @@
   "world.reward.building.title": "{0} a été complété(e)",
   "world.meet.tribe": "Vous avez rencontré {0}",
   "world.task.new": "Vous avez une nouvelle tâche !",
-
   "tribes.nature": "Nature",
   "tribes.xin-xi": "Xin-xi",
-  "tribes.xin-xi.info": "Ils commencent leur voyage dans les montagnes denses, entourés de magnifiques fleurs de cerisier.\n\nLes {0} commencent la partie avec la technologie '{1}'.",
+  "tribes.xin-xi.info": "Ils commencent leur voyage dans les montagnes denses, entourée de magnifiques fleurs de cerisier.\n\nLes {0} commencent la partie avec la technologie '{1}'.",
```

Comment:
"entourés" or maybe "entourées" but not "entourée". The noun can be "ils" or "les montagnes". It sounds like it could be more right if it's "les montagnes". I can change it. What do you think about it?